apps/configurator/lib/configurator_web/controllers/map_configuration_controller.ex

```diff
@@ -63,11 +63,72 @@ defmodule ConfiguratorWeb.MapConfigurationController do
     |> redirect(to: ~p"/map_configurations")
   end
 
+  def edit_obstacles(conn, %{"id" => id}) do
```

Comment:
Thanks for this, but I think it's odd to have two ways of editing the same thing. Let's create an issue to integrate these forms into the map form or to refactor the map form entirely.

Reply:
Hmm yeah it was a but weird but i don't think we should lose the json field, al least for now, i just added these buttons to the map configuration edit so they are more accesible